---
permalink: /
title: "Home"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---
Welcome to my homepage!

I am currently a Research Assistant Professor in [Department of Pathology and Laboratory Medicine](https://pathology.med.upenn.edu) 
at the [University of Pennsylvania](https://www.upenn.edu) (Penn),
additionally affiliated with [Center for Computational and Genomic Medicine](https://www.research.chop.edu/center-for-computational-and-genomic-medicine) 
at the [Children's Hospital of Philadelphia Research Institute](https://www.research.chop.edu) (CHOP).

Previously, I obtained my Ph.D. in Biostatistics from [Department of Biostatistics](https://sph.unc.edu/bios/biostatistics/) at the [University of North Carolina at Chapel Hill (UNC-CH)](https://www.unc.edu),
advised by [Dr. Yun Li](https://sph.unc.edu/adv_profile/yun-li-phd/). I also worked closely with 
[Dr. Nora Franceschini](https://sph.unc.edu/adv_profile/nora-franceschini-md/),
[Dr. Christopher Sayed](https://www.med.unc.edu/derm/directory/christopher-sayed-md/)
and [Dr. Laura Raffield](https://www.med.unc.edu/genetics/directory/laura-raffield-phd/). 
Before that, I received my B.S. in Mathematics and Applied Mathematics from the [School of Mathematical Sciences](http://www.math.zju.edu.cn/mathen/) at [Zhejiang University](https://www.zju.edu.cn/english/).


Research Briefings
======
My research has been focusing on statistical genetics and genomics, 
including both developing statistical methods applicable to genetics and genomics fields, 
and analyzing large-scale biomedical data to disentangle complex diseases and traits,
including a special focus on promoting diversity in genetics studies. 
Specifically, I have developed statistical methods and computational tools for 
exploring linkage disequilibrium (LD) with whole genome sequencing (WGS) data from [Trans-Omics for Precision Medicine (TOPMed) Project](https://topmed.nhlbi.nih.gov)
([TOP-LD](http://topld.genetics.unc.edu),
improving genotype imputation quality calibration
(e.g., [MagicalRsq](https://pubmed.ncbi.nlm.nih.gov/36198314/), [MagicalRsq-X](https://pubmed.ncbi.nlm.nih.gov/38636510)),  
performing genetics association testing (e.g., [eSCAN](https://pubmed.ncbi.nlm.nih.gov/34882196/)), 
improving polygenic risk score (PRS) prediction accuracy
(e.g., [GAUDI](https://pubmed.ncbi.nlm.nih.gov/38310129/), [EndoPRS](https://pubmed.ncbi.nlm.nih.gov/40203832/)),
inferring cell-type-specific gene expression profile from bulk data
(e.g., [EPIC-unmix](https://www.biorxiv.org/content/10.1101/2024.05.23.595514v1)) 
and integrating multi-omics data for risk prediction and biological interpretation.
(e.g., [PEARL](https://www.biorxiv.org/content/10.1101/2025.05.19.654754v1.abstract), ([VAMPIRE](https://pubmed.ncbi.nlm.nih.gov/35047852/))). 
After joining CHOP/Penn, I have been working with [Dr. Yi Xing](https://www.research.chop.edu/people/yi-xing) on developing statistical models
and computational pipelines to analyze RNA splicing powered by long-read RNA sequencing (lrRNA-seq) data, 
as well as integrating with genetics data to uncover phenotype-transcriptomics-genomics relationships.

Additionally, I have also been participating in genotype imputation and large-scale genome-wide association studies (GWAS), 
and have identified several novel signals that may guide future functional genomics studies and eventually impact clinical practices.
I have extensive experience in a broad spectrum of traits and diseases, including blood cell traits, kidney traits, chronic kidney disease (CKD), 
sickle cell disease, type II diabetes, cardiovascular diseases, hidradenitis suppurativa (HS), and cystic fibrosis (CF).

I have been strongly advocate to promote inclusion in genetic epidemiological studies to include global populations.
For example, I have recently published a [review article](https://pubmed.ncbi.nlm.nih.gov/40185073/) on 
opportunities and challenges incorporating local ancestry in genetic association studies.
I am also actively involved in multiple studies for non-European populations, including the 
[Jackson Heart Study](https://www.jacksonheartstudy.org) of African American individuals, 
and the [Strong Heart Study](https://strongheartstudy.org) of American Indian individuals.
Further, I am also a member of several multi-institution consortium efforts, including the [TOPMed project](https://topmed.nhlbi.nih.gov), 
[Cystic Fibrosis Genome Project (CFGP)](https://www.cff.org/researchers/whole-genome-sequencing-project-data-requests), 
[Polygenic Risk Methods in Diverse Populations (PRIMED) consortium](https://primedconsortium.org),
[Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium](https://www.chargeconsortium.com) 
and [Impact of Genomic Variation on Function (IGVF) consortium](http://igvf.org). 

**Key words**

  * Statistical genetics and genomics
  * Bioinformatics and computational biology
  * Polygenic risk score
  * Global population health
  * Multi-omics
  * High-dimensional data
  * Machine learning and deep learning





