---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

List in [Google Scholar](https://scholar.google.com/citations?user=McM4hl0AAAAJ&hl=en)

Last update: 5/16/2024

<sup>*</sup> denotes co-first authorship.
\# denotes co-corresponding authorship.

## Pre-print & Submitted

1. (Submitted) Tang Y, Du J, Best LG, Haack K, Newman D, Zhang Y, Cole SA, **Sun Q**\#, Franceschini N\#.
Transferability of Polygenic Scores for C-reactive Protein to American Indians: The Strong Heart Study. 2024+.

1. (Submitted) **Sun Q**<sup>*</sup>, Karafin MS<sup>*</sup>, Garrett ME, Li Y, Ashley-Koch A, Telen MJ.
A genome-wide association study of alloimmunization in the TOPMed OMG-SCD cohort identifies a locus on chromosome 12. 2024+. 

1. (Submitted) Chen J, Xiong C, **Sun Q**, Gupta G, Halder A, Li D, Li Y. 
Investigating spatial dynamics in spatial omics data with StarTrail. 2024+.

1. (Submitted) Nagarajan P, Winkler T, Bentley A, Miller C, Kraja A, Schwander K, ..., **Sun Q**, ..., Noordam R, Wang H.
A Large-Scale Genome-Wide Study of Gene-Sleep Duration Interactions for Blood Pressure in 811,405 Individuals from Diverse Populations.
2024+.
[medRxiv 2024.03.07.24303870](https://doi.org/10.1101/2024.03.07.24303870).

1. (Submitted) Sun Z, Yi Z, Wei C, Wang W, Cravedi P, Tedla F, Ward SC, ..., **Sun Q**, ..., Zhang W.
Genetic polymorphisms of Leukocyte Immunoglobulin-Like Receptor B3 _LILRB3_ gene in African American kidney transplant recipients are associated with post-transplant graft failure. 
2024+.

1. (Submitted) Yang X<sup>*</sup>, Jones lR<sup>*</sup>, Chen PB<sup>*</sup>, Yang H<sup>*</sup>, Ren X<sup>*</sup>, Zheng L, Li B, Li YE, **Sun Q**, Wen J, Beaman C, Cui X, Li Y, Wang W, Hu M, Ren B, Shen Y. 
Functional characterization of gene regulatory elements and neuropsychiatric disease-associated risk loci in iPSCs and iPSC-derived neurons. 2023+. 
[bioRxiv 2023.08.30.555359](https://doi.org/10.1101/2023.08.30.555359).

1. (Submitted) Liu S, Zhu J, Zhong H, Wu C, Xue H, Darst BF, ..., **Sun Q**, ..., Rich SS, Rotter JI, Wu L.
Identification of proteins associated with Type 2 diabetes risk in racial and ethnic diverse populations. 2023+.

1. (Submitted) Wen J, Lin B, **Sun Q**, Jiang M, Go A, Miller-Fleming T, Shuey M, Cohen D, Rao P, Rahman M, Cox N, Lash J, Franceschini N, Li Y.
Genetics of cardiovascular outcomes in individuals with chronic kidney disease: the Chronic Renal Insufficiency Cohort (CRIC) study. 2023+.

1. (Submitted) Wen J<sup>*</sup>, **Sun Q**<sup>*</sup>, Huang L, Zhou L, Doyle MF, Ekunwe L, Olson NC, Reiner AP, Li Y, Raffield LM. 
Gene Expression and Splicing QTL Analysis of Blood Cells in African American Participants from the Jackson Heart Study. 2023+.
[bioRxiv 2023.04.26.538455](https://doi.org/10.1101/2023.04.26.538455).

1. (Submitted) Wen J, Li G, Chen J, **Sun Q**, Liu W, Guan W, Lai B, Szatkiewicz JP, He X, Sullivan PF, Li Y.
DeepGWAS: Enhance GWAS Signals for Neuropsychiatric Disorders via Deep Neural Network. 2022+. 
[bioRxiv 2022.12.20.521277](https://doi.org/10.1101/2022.12.20.521277).

1. (Submitted) Horimoto ARVR, **Sun Q**, Browning S, Lash JP, Cai J, Haack K, Cole SA, Thornton TA, Franceschini N.
Admixture mapping screening of CKD traits and risk factors in U.S. Hispanic individuals from Central America country of origin. 2022+. 
[medRxiv 2022.06.17.22276554](https://doi.org/10.1101/2022.06.17.22276554)
{: reversed="reversed"}

## Published

1. Best LG, Erdei E, Haack K, Kent JW, Malloy K, Newman DE, O'Leary MA, O'Leary RA, **Sun Q**, Navas-Acien A, Franceschini N, Cole SA.
Genetic variant rs1205 associated with COVID-19 Outcomes: The Strong Heart Study and Strong Heart Family Study.
_PLoS One._ 2024;19(4):e0302464. Published 2024 Apr 25.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/38662664/);
[Full-Text](https://doi.org/10.1371/journal.pone.0302464).

1. **Sun Q**, Yang Y, Rosen JD, Chen J, Li X, Guan W, Jiang M, Wen J, Pace RG, Blackman SM, Bamshad MJ, Gibson RL, Cutting GR, O'Neal WK, Knowles MR, Kooperberg C, Reiner AP, Raffield LM, Carson AP, Rich SS, Rotter JI, Loos RJF, Kenny E, Jaeger BC, Min YI, Fuchsberger C, Li Y.
MagicalRsq-X: A cross-cohort transferable genotype imputation quality metric.
_Am J Hum Genet._ 2024;111(5):990-995. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/38636510/);
[Full-Text](https://doi.org/10.1016/j.ajhg.2024.04.001);
[GitHub](https://github.com/quansun98/MagicalRsqX);
[News](https://www.ashg.org/ajhg/inside-ajhg-with-yun-li-and-quan-sun/).

1. Hou K, Gogarten S, Kim J, Hua X, Dias J, **Sun Q**, Wang Y, Tan T, Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium Methods Working Group,
Atkinson EG, Martin A, Shortt J, Hirbo J, Li Y, Pasaniuc B, Zhang H.
Admix-kit: An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations.
_Bioinformatics._ 2024;40(4):btae148. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/38490256/);
[Full-Text](https://doi.org/10.1093/bioinformatics/btae148);
[GitHub](https://github.com/KangchengHou/admix-kit).

1. **Sun Q**<sup>*</sup>, Rowland BT<sup>*</sup>, Chen J, Mikhaylova AV, Avery C, Peters U, Lundin J, Matise T, Buyske S, Tao R, Mathias RA, Reiner AP, Auer PL, Cox NJ, Kooperberg C, Thornton TA, Raffield LM, Li Y.
Improving polygenic risk prediction in admixed populations by explicitly modeling ancestral-specific effects via GAUDI. 
_Nat Commun._ 2024;15(1):1016.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/38310129/);
[Full-Text](https://www.nature.com/articles/s41467-024-45135-z);
[GitHub](https://github.com/quansun98/GAUDI).

1. Jeng XJ, Hu Y, **Sun Q**, Li Y. 
Weak signal inclusion under dependence and applications in genome-wide association study.
_Ann. Appl. Stat._ 2024;18(1):841-857.
[Abstract](https://projecteuclid.org/journals/annals-of-applied-statistics/volume-18/issue-1/Weak-signal-inclusion-under-dependence-and-applications-in-genome-wide/10.1214/23-AOAS1815.short)

1. Hughes O, Bentley AR, Breeze C, Aguet F, Xu X, Nadkarni G, **Sun Q**, Lin B, Gilliland T, ..., Rotimi C, Morris A, Franceschini N.
Genome-wide association study for kidney function in individuals of African ancestry and ancestry in the Americas provides insight into effector genes and polygenic prediction.
_Cell Genomics._ 2023;100468.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/38190104/);
[Full-Text](https://www.sciencedirect.com/science/article/pii/S2666979X23003129?via%3Dihub).

1. **Sun Q**<sup>*</sup>, Rowland BT<sup>*</sup>, Wang W, Miller-Fleming T, Cox NJ, Graff M, Faucon A, Shuey MM, Blue EE, Auer PL, Li Y, Sankaran VG, Reiner AP, Raffield LM. 
Genetic Examination of Hematological Parameters in SARS-CoV-2 Infection and COVID-19. 
_Blood Cells Mol Dis._ 2023;103:102782. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/37558590/);
[Full-Text](https://www.sciencedirect.com/science/article/pii/S1079979623000591?via%3Dihub).

1. **Sun Q**<sup>*</sup>, Broadaway KA<sup>*</sup>, Edmiston SN, Fajgenbaum K, Miller-Fleming T, Westerkam LL, Melendez-Gonzalez M, Bui  H, Blum FR, Levitt B, Lin L, Hao H, Harris KM, Liu Z, Thomas NE, Cox NJ, Li Y, Mohlke KL, Sayed CJ. 
Genetic Variants Associated With Hidradenitis Suppurativa. 
_JAMA Dermatol._ 2023;e232217.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/37494057/);
[Full-Text](https://jamanetwork.com/journals/jamadermatology/article-abstract/2807701);
[News](https://www.med.unc.edu/genetics/researchers-discover-genetic-locations-for-increased-risk-of-hidradenitis-suppurativa/).

1. Huerta-Chagoya A, Schroeder P, Mandla R, Deustch A, ..., **Sun Q**, ..., Florez JC, Leong A, Mercader JM. 
The power of TOPMed imputation for the discovery of Latino-enriched rare variants associated with type 2 diabetes.
_Diabetologia._ 2023;66(7):1273-1288.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/37148359/);
[Full-Text](http://doi.org/10.1007/s00125-023-05912-9).

1. Sullivan PF, Meadows JRS, Gazal S,  Phan BN, Li X, Genereux DP, Dong MX, Bianchi M, Andrews G, Sakthikumar S, Nordin J, Roy A, Christmas MJ, Marinescu VD, Wang C, Wallerman O, Xue J, Yao S, **Sun Q**, ..., Karlsson EK, Lindblad-Toh K.
Leveraging Base Pair Mammalian Constraint to Understand Genetic Variation and Human Disease. 
_Science._ 2023;380(6643):eabn2937.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/37104612/);
[Full-Text](http://doi.org/10.1126/science.abn2937).

1. Zhou YH, Gallins PJ, Pace RG, Dang H, Aksit MA, Blue EE, Buckingham KJ, Collaco JM, Faino AV, Gordon WW, Hetrick KN, Ling H, Liu W, Onchiri FM, Pagel K, Pugh EW, Raraigh KS, Rosenfeld M, **Sun Q**, Wen J, Li Y, Corvol H, Strug LJ, Bamshad MJ, Blackman SM, Cutting GR, Gibson RL, O'Neal WK, Wright FA, Knowles MR. 
Genetic Modifiers of Cystic Fibrosis Lung Disease Severity: Whole Genome Analysis of 7,840 Patients. 
_Am J Respir Crit Care Med._ 2023;207(10):1324-1333. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/36921087/);
[Full-Text](http://doi.org/10.1164/rccm.202209-1653OC).

1. Thompson KA, Hedlund EL, **Sun Q**, Peat CM, Goode RW, Termorshuizen JD, Thornton LM, Borg S, var Furth ER, Birgegård A, Bulik CM, Watson HJ. 
Course and predictors of eating disorder symptoms, anxiety symptoms, and pandemic-related eating disorder concerns among adults with eating disorders during the first year of the COVID-19 pandemic.
_Int J Eat Disord._ 2023;56(1):151-168. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/36508290/);
[Full-Text](http://doi.org/10.1002/eat.23870).

1. Termorshuizen JD, **Sun Q**, Borg S, Mantilla EF, Goode RW, Peat CM, Thornton LM, Watson HJ, var Furth ER, Birgegård A, Bulik CM.
Longer-term impact of COVID-19 among individuals with self-reported eating disorders in the United States, the Netherlands, and Sweden.
_Int J Eat Disord._ 2023;56(1):80-90.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/36237128/);
[Full-Text](http://doi.org/10.1002/eat.23824).

1. Aksit MA, Ling H, Pace RG, Raraigh KS, Onchiri F, Faino AV, Pagel K, Pugh E, Stilp AM, **Sun Q**, Wright FA, Zhou Y, Bamshad MJ, Gibson RL, Knowles MR, Cutting GR, Blackman SM, CF Genome Project. 
Pleiotropic modifiers of age-related diabetes and neonatal intestinal obstruction in cystic fibrosis. 
_Am J Hum Genet._ 2022;109(10):1894-1908.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/36206743/);
[Full-Text](https://doi.org/10.1016/j.ajhg.2022.09.004).

1. **Sun Q**, Yang Y, Rosen JD, Jiang MZ, Chen J, Liu W, Wen J, Raffield LM, Pace RG, Zhou YH, Wright FA, Blackman SM, Bamshad MJ, Gibson RL, Cutting GR, Knowles MR, Schrider DR, Fuchsberger C, Li Y. 
Machine-learning-based genotype imputation quality calibration. 
_Am J Hum Genet._ 2022;109(11):1986-1997.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/36198314/);
[Full-Text](https://doi.org/10.1016/j.ajhg.2022.09.009);
[GitHub](https://github.com/quansun98/MagicalRsq);
[News](https://www.med.unc.edu/genetics/yun-lis-lab-publishes-new-paper-in-ajhg/).

1. Zhong W<sup>*</sup>, Liu W<sup>*</sup>, Chen J, **Sun Q**, Hu M, Li Y.
Understanding the function of regulatory DNA interactions in the interpretation of non-coding GWAS variants.
_Front Cell Dev Biol._ 2022;10:957292.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/36060805/);
[Full-Text](https://doi.org/10.3389/fcell.2022.957292).

1. Huang L<sup>*</sup>, Rosen JD<sup>*</sup>, **Sun Q**<sup>*</sup>, Chen J, Wheeler MM, Zhou Y, Min YI, Kooperberg C, Conomos MP, Stilp AM, Rich SS, Rotter JI, Manichaikul A, Loos RJF, Kenny EE, Blackwell TW, Smith AV, Jun G, Sedlazeck FJ, Metcalf G,  Boerwinkle E, NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium, Raffield LM, Reiner AP, Auer PL, Li Y.
TOP-LD: A tool to explore linkage disequilibrium with TOPMed whole-genome sequence data.
_Am J Hum Genet._ 2022;109(6):1175-1181. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/35504290/);
[Full-Text](https://doi.org/10.1016/j.ajhg.2022.04.006);
[Website](http://topld.genetics.unc.edu).

1. Zhao B, Li T, Smith SM, Xiong D, Wang X, Yang Y, Luo T, Zhu Z, Shan Y, Matoba N, **Sun Q**, Yang Y, Hauberg ME, Bendl J, Fullard JF, Roussos P, Lin W, Li Y, Stein JL, Zhu H.
Common variants contribute to intrinsic human brain functional networks.
_Nat Genet._ 2022;54(4):508-517. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/35393594/);
[Full-Text](https://doi.org/10.1038/s41588-022-01039-6).

1. Granot-Hershkovitz E, **Sun Q**, Argos M, Zhou H, Lin X, Browning SR, Sofer T.
AFA: Ancestry-specific allele frequency estimation in admixed populations: The Hispanic Community Health Study/Study of Latinos. 
_HGG Adv._ 2022;3(2):100096.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/35300209/);
[Full-Text](https://doi.org/10.1016/j.xhgg.2022.100096).

1. Liu W, **Sun Q**, Huang L, Bhattacharya A, Wang GW, Tan X, Kuban KCK, Joseph RM, O'Shea TM, Fry RC, Li Y, Santos HP Jr. 
Innovative computational approaches shed light on genetic mechanisms underlying cognitive impairment among children born extremely preterm.
_J Neurodev Disord._ 2022;14(1):16.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/35240980/);
[Full-Text](https://doi.org/10.1186/s11689-022-09429-x).

1. Rowland B, Venkatesh S, Tardaguila M, Wen J, Rosen JD, Tapia AL, **Sun Q**, Graff M, Vuckovic D, Lettre G, Sankaran VG, Voloudakis G, Roussos P, Huffman JE, Reiner AP, Soranzo N, Raffield LM, Li Y. 
Transcriptome-wide association study in UK Biobank Europeans identifies associations with blood cell traits. 
_Hum Mol Genet._ 2022;31(14):2333-2347. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/35138379/);
[Full-Text](https://doi.org/10.1093/hmg/ddac011).

1. Wen J<sup>*</sup>, Lagler TM<sup>*</sup>, **Sun Q**<sup>*</sup>, Yang Y, Chen J, Harigaya Y, Sankaran VG, Hu M, Reiner AP, Raffield LM, Li Y. 
Super interactive promoters provide insight into cell type-specific regulatory networks in blood lineage cell types. 
_PLoS Genet._ 2022;18(1):e1009984. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/35100265/);
[Full-Text](https://doi.org/10.1371/journal.pgen.1009984).

1. **Sun Q**<sup>*</sup>, Liu W<sup>*</sup>, Rosen JD, Huang L, Pace RG, Dang H, Gallins PJ, Blue EE, Ling H, Corvol H, Strug LJ, Bamshad MJ, Gibson RL, Pugh EW, Blackman SM, Cutting GR, O'Neal WK, Zhou Y, Wright FA, Knowles MR, Wen J, Li Y. 
Leveraging TOPMed imputation server and constructing internal imputation reference panels to enhance genotype imputation among individuals affected with cystic fibrosis.
_HGG Adv._ 2022;3(2):100090. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/35128485/);
[Full-Text](https://doi.org/10.1016/j.xhgg.2022.100090).

1. Yang Y<sup>*</sup>, **Sun Q**<sup>*</sup>, Huang L, Broome JG, Correa A, Reiner A; NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium, Raffield LM, Yang Y, Li Y.
eSCAN: scan regulatory regions for aggregate association testing using whole-genome sequencing data.
_Brief Bioinform._ 2022;23(1):bbab497.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/34882196/);
[Full-Text](https://doi.org/10.1093/bib/bbab497).

1. **Sun Q**<sup>*</sup>, Crowley CA<sup>*</sup>, Huang L, Schrider D, Auer P, Lettre G, Sankaran V, Reiner A, Raffield LM, Li Y.
From GWAS variant to function: A study of ∼148,000 variants for blood cell traits.
_HGG Adv._ 2021;3(1):100063.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/35047852/);
[Full-Text](https://doi.org/10.1016/j.xhgg.2021.100063).

1. Mikhaylova AV, McHugh CP, ..., **Sun Q**, ..., Auer PL. 
Whole-genome sequencing in diverse subjects identifies genetic correlates of leukocyte traits: The NHLBI TOPMed program.
_Am J Hum Genet._ 2021;108(10):1836-1851.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/34582791/);
[Full-Text](https://doi.org/10.1016/j.ajhg.2021.08.007).

1. Little A, Hu Y, **Sun Q**, Jain D, ..., Li Y, Auer PL, Thornton T, Reiner AP, Johnson AD, Raffield LM. 
Whole genome sequence analysis of platelet traits in the NHLBI trans-omics for precision medicine initiative. 
_Hum Mol Genet._ 2022;31(3):347-361.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/34553764/);
[Full-Text](https://doi.org/10.1093/hmg/ddab252).

1. Wen J, Xie M, Rowland B, Rosen JD, **Sun Q**, Chen J, Tapia AL, Qian H, Kowalski MH, Shan Y, Young KL, Graff M, Argos M, Avery CL, Bien SA, Buyske S, Yin J, Choquet H, Fornage M, Hodonsky CJ, Jorgenson E, Kooperberg C, Loos RJF, Liu Y, Moon JY, North KE, Rich SS, Rotter JI, Smith JA, Zhao W, Shang L, Wang T, Zhou X, Reiner AP, Raffield LM, Li Y.
Transcriptome-Wide Association Study of Blood Cell Traits in African Ancestry and Hispanic/Latino Populations. 
_Genes (Basel)._ 2021;12(7):1049.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/34356065/);
[Full-Text](https://doi.org/10.3390/genes12071049).

1. **Sun Q**, Graff M, Rowland B, Wen J, Huang L, Miller-Fleming TW, Haessler J, Preuss MH, Chai JF, Lee MP, Avery CL, Cheng CY, Franceschini N, Sim X, Cox NJ, Kooperberg C, North KE, Li Y, Raffield LM. 
Analyses of biomarker traits in diverse UK biobank participants identify associations missed by European-centric analysis strategies. 
_J Hum Genet._ 2022;67(2):87-93.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/34376796/);
[Full-Text](https://doi.org/10.1038/s10038-021-00968-0).

1. Yu M, Abnousi A, Zhang Y, Li G, Lee L, Chen Z, Fang R, Lagler TM, Yang Y, Wen J, **Sun Q**, Li Y, Ren B, Hu M. 
SnapHiC: a computational pipeline to identify chromatin loops from single-cell Hi-C data. 
_Nat Methods._ 2021;18(9):1056-1059.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/34446921/);
[Full-Text](https://doi.org/10.1038/s41592-021-01231-2).

1. Hu Y, Stilp A, ..., **Sun Q**, ..., Reiner A, NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium.
Whole genome sequencing association analysis of quantitative red blood cell phenotypes: the NHLBI TOPMed program [published correction appears in _Am J Hum Genet._ 2021 Jun 3;108(6):1165].
_Am J Hum Genet._ 2021;108(5):874-893.
[PubMed](https://pubmed.ncbi.nlm.nih.gov/33887194/);
[Full-Text](https://doi.org/10.1016/j.ajhg.2021.04.003).

1. Hu Y, Bien SA, ..., **Sun Q**, ..., Kooperberg C.
Multi-ethnic genome-wide association analyses of white blood cell and platelet traits in the Population Architecture using Genomics and Epidemiology (PAGE) study [published correction appears in _BMC Genomics._ 2021 Sep 13;22(1):656].
_BMC Genomics._ 2021;22(1):432. 
[PubMed](https://pubmed.ncbi.nlm.nih.gov/34107879/);
[Full-Text](https://doi.org/10.1186/s12864-021-07745-5).
{: reversed="reversed"}




